concent_api/verifier/tasks.py

```diff
@@ -106,264 +107,252 @@ def blender_verification_order(
         operation=message.FileTransferToken.Operation.download,
     )
 
-    # Remove any files from VERIFIER_STORAGE_PATH.
-    clean_directory(settings.VERIFIER_STORAGE_PATH)
```

Comment:
There was a requirement to clean directory at the beginning of the task. I know that now all exceptions are caught and directory is cleaned there, but I think it is still possible for verification to be stopped without this (ie. machine get powered off). So I think this should stay.